### ISSUE TITLE
Error when not using any plugins

### ISSUE THREAD

**Issue by beyerleinf** [beyerleinf/esbuild-azure-functions#7]
`write` should always be set to false.

```
TypeError: result.outputFiles is not iterable
    at build (/Users/beyerleinf/**********/node_modules/esbuild-azure-functions/build/src/lib/builder.js:52:31)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async file:///Users/beyerleinf/**********/build.mjs:11:1
```

**Comment by jeremiah-carlin** [comment]
I just found your project and it solved a nasty set of issues around deploying a pnpm monorepo to a Azure static web app.  

I hit the error above but it seemed like a warning because the esbuild bundle completed and I was able to run the functions.  I was able to work around the warnings by adding `write: false` to the esbuildOptions, though the azure functions were deployable even with the warning error that showed up.

Thank you for writing this.  Its a really elegant solutions to dealing with node azure functions.
